properties.py

```diff
@@ -654,10 +649,6 @@ def reset_ll_object_index(self, context):
         name="Max Diffuse Depth",
         description="Maximum number of diffuse ray bounces",
         min=0, max=32, default=1)
-    use_separate_path_depths = BoolProperty(
```

Comment:
Yes. According to the release notes for 21 it does this by default now anyway.

Sent from my iPhone

On Jul 23, 2016, at 12:59 PM, bsavery <notifications@github.com<mailto:notifications@github.com>> wrote:


In properties.py<https://github.com/bsavery/PRMan-for-Blender/pull/315#discussion_r71975846>:

> @@ -654,10 +649,6 @@ def reset_ll_object_index(self, context):
>          name="Max Diffuse Depth",
>          description="Maximum number of diffuse ray bounces",
>          min=0, max=32, default=1)
> -    use_separate_path_depths = BoolProperty(


we're taking this out now?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/bsavery/PRMan-for-Blender/pull/315/files/87a9e615ca048d85d6ed1e0db290f7b96aa8f5e1#r71975846>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AM5RFEy7o3ENCBLKr4ystQJqM-BnxHH9ks5qYlaHgaJpZM4JTXn2>.